CHANGES.md

```diff
@@ -17,6 +17,7 @@
   is deployed as a k8s daemonset.
 * Add `consistencyMode` option to `io.l5d.consul` namer
 * Add `readConsistencyMode` and `writeConsistencyMode` options to `io.l5d.consul` dtab storage
+* Add `failFast` (defaults to `false`) to consul namer and dtab storage options
```

Comment:
Hmmm... this is a change in the default behavior so I might word it to emphasize that.  Something like "failFast is now disabled by default for the consul namer and dtab storage but can be enabled with the `failFast`option"